config/storage/ibm_cos.md

```diff
@@ -34,8 +34,7 @@ Lithops with IBM COS as storage backend.
         storage_backend: ibm_cos
        
     ibm_cos:
-       endpoint   : <REGION_ENDPOINT>  
-       private_endpoint : <PRIVATE_REGION_ENDPOINT>
```

Comment:
@JosepSampe add them back. May be put in comment the usage, etc ?

Reply:
@gilv Like in the rest of compute and storage backends, this config example intend to show minimum configuration needed. All available keys are specified in the table below. If we add the keys here, we can delete the table, I mean, there is no need to duplicate things.